modules/apps/29-fee/types/errors.go

```diff
@@ -16,4 +16,5 @@ var (
 	ErrFeeNotEnabled                 = sdkerrors.Register(ModuleName, 9, "fee module is not enabled for this channel. If this error occurs after channel setup, fee module may not be enabled")
 	ErrRelayerNotFoundForAsyncAck    = sdkerrors.Register(ModuleName, 10, "relayer address must be stored for async WriteAcknowledgement")
 	ErrFeeModuleLocked               = sdkerrors.Register(ModuleName, 11, "the fee module is currently locked, a severe bug has been detected")
+	ErrDistributionAddressEmpty      = sdkerrors.Register(ModuleName, 12, "distribution address must not be empty")
```

Comment:
Also a nit: since we haven't released fee yet, should we put this error next to `ErrCounterpartyAddressEmpty`?

Reply:
Removed this error as its obsolete now that we check the `payee` field directly using `sdk.AccAddressFromBech32` in `ValidateBasic`.